Update documentation to add ingress and egress filters (#2771)
hashicorp/boundary: website/content/docs/concepts/filtering/worker-tags.mdx

# Target Worker Filtering

> Once workers have tags, these tags may be used to control which

Reviewer: ```suggestion
Once workers have tags, you can use these tags to control which
```

Author: thanks!

workers are allowed to manage a given session by specifying optional worker filter attributes

> when configuring [targets](/docs/concepts/domain-model/targets).

Reviewer: ```suggestion
when you configure [targets](/docs/concepts/domain-model/targets).
```

> The `egress_worker_filter` attribute controls which workers will be used for egress to a target. This is the worker
> that will access the target.

Reviewer: ```suggestion
The `egress_worker_filter` attribute controls which workers are used for egress to a target. This is the worker
that accesses the target.
```

Author: thanks!

> The `ingress_worker_filter`<sup>HCP Only</sup> attribute controls which workers will be used for ingress to a target.

Reviewer: ```suggestion
The `ingress_worker_filter`<sup>HCP Only</sup> attribute controls which workers are used for ingress to a target.
```

> This is the worker a client will connect to when initiating a connection.

Reviewer: ```suggestion
This is the worker a client connects to when initiating a connection.
```

Reviewer: This sentence sounds a little funny to me with the repetition of the verb "to connect". I'm wondering if we can be more specific. Would it be accurate to say:

> This is the worker a client connects to when initiating a connection **to a target**.

Author: I like that! Thanks!
